extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -416,19 +416,30 @@ export class AmpConsent extends AMP.BaseElement {
     this.passSharedData_();
     this.maybeSetDirtyBit_();
 
-    this.getConsentRequiredPromise_()
-      .then(isConsentRequired => {
-        return this.initPromptUI_(isConsentRequired);
-      })
-      .then(isPostPromptUIRequired => {
-        if (isPostPromptUIRequired) {
-          this.handlePostPromptUI_();
-        }
+    this.consentStateManager_.getLastConsentInstanceInfo().then(storedInfo => {
+      if (
+        isExperimentOn(this.win, 'amp-consent-geo-override') &&
```

Comment:
can this be moved into `getConsentRequiredPromise_`?

Reply:
Good point